.github/workflows/release-deploy.yml

```diff
@@ -32,3 +35,19 @@ jobs:
       with:
         user: __token__
         password: ${{ secrets.PYPI_TOKEN_VALIDATOR }}
+
+    - name: Build Windows executables
+      if: matrix.os == 'windows-latest'
+      run: |
+        python -m pip install -r requirements-dev.txt
+        pyinstaller dicom-validator.spec -y
+
+    - name: Publish Windows executables on GitHub
+      if: matrix.os == 'windows-latest'
+      uses: softprops/action-gh-release@v2
+      env:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      with:
+        files: |
+          - dist/validate_iods.exe
```

Comment:
I'm not sure how to test this except for an actual release. But even if that fails, it would not be a big deal...

Reply:
I ran into the same. That is why I kept the second workflow on (which tests this for a pull request)